### DISCUSSION TITLE
LSASS Process utilization

### DISCUSSION THREAD

**Discussion by mtace** [datalust/seq-tickets#2005] [Help]
Is it normal for the SEQ to utilize the LSASS (Local Security Authority Subsystem Service) of Windows?
Mostly there is a constant CPU utilization of the SEQ service <10%. LSASS is about the same, most times much higher. When I stop the SEQ service LSASS CPU utilization drops to zero.
Sometimes SEQ 0,1% and SEQ 10%. Depending on the amount of data to log.

Best regards

**Comment by nblumhardt** [comment]
Hi @mtace; Seq uses LSASS (as far as I understand it) when validating credentials for Active Directory authentication, but I'm not used to seeing any significant LSASS CPU usage in conjunction with running Seq.

Are you running any Seq apps that might be interacting with it? Or is your Seq sever also your AD domain controller? (Looking at https://learn.microsoft.com/en-us/troubleshoot/windows-server/identity/troubleshoot-high-lsass.exe-cpu-utilization.)
